plugins/lb302/lb302.cpp

```diff
@@ -229,7 +229,7 @@ void lb302Filter3Pole::envRecalc()
 	w = vcf_e0 + vcf_c0;
 	k = (fs->cutoff > 0.975)?0.975:fs->cutoff;
 	kfco = 50.f + (k)*((2300.f-1600.f*(fs->envmod))+(w) *
-	                   (700.f+1500.f*(k)+(1500.f+(k)*(Engine::mixer()->processingSampleRate()/2.f-6000.f)) *
+	                   (700.f+1500.f*(k)+(1500.f+(k)*(44100.f/2.f-6000.f)) *
```

Comment:
See comment above.

Reply:
Yes, a named constant would be better.
It most likely should be a constant. What happens when the sampling rate changes is that the filters cutoff frequency changes for the same settings otherwise, compare the figures below. The `w` is constant btw, see line 212.

![CutoffFreq](https://user-images.githubusercontent.com/64359888/89819849-55b2cc80-db4c-11ea-9d6a-37399fc944f6.png)

Reply:
Thanks for the explanation there. It seems really odd at face value, but DSP still doesn't come naturally to me, especially filter coefficients and such. I think it's fine as long as it's easy to see that `44100.0f` is intentional.

Reply:
That is just the `kfco` in the code, for which I think it is at least approximately the cutoff frequency at the start of a note, plotted against the knob position. I just noticed that if the plots above are valid then only for short notes. The `vcf_c0` in `w = vcf_e0 + vcf_c0` is set to `1` at the beginning of a note (for which I generated the plots) and then gets successively multiplied with the decay `vcf_c0 *= fs->envdecay` in `lb302filter::envRecalc()`. After a few moments `w` is nearly zero and the `kfco` reduces to `50.f + (k)*((2300.f-1600.f*(fs->envmod))` so the Env Mod knob is decreasing the cutoff now instead of increasing it like at the beginning of a note. You can hear this very well when you adjust the Env Mod knob for a pattern of many short notes versus a single long note. 

Anyways,  this does not change my opinion that there shouldn't be a dependency of `kfco` on the dynamic sampling rate, which changes the filters behavior completely for the same knob positions. What do you think?